packages/services/service-common/src/helpers.ts

```diff
@@ -33,6 +33,9 @@ export function invariant(
   throw new Error(value);
 }
 
-export function maskToken(token: string) {
+export function maskToken(token: string): string {
```

Comment:
```
maskToken('')
// Uncaught RangeError: Invalid count value: -1
//   at String.repeat (<anonymous>)
```

Reply:
@TuvalSimha we can return the string as is if the length is below a threshold (e.g. 6 characters) in that case it is not a valid token and thus does not require masking.

Reply:
i agree with @n1ru4l